fixtures/topology/k8s-system.yaml

```diff
@@ -44,15 +44,32 @@ spec:
                 name: k8s
                 display:
                   javascript: JSON.stringify(k8s.getNodeTopology(results)) 
-          properties:
-            - name: node-metrics
-              lookup:
-                kubernetes:
-                  - kind: NodeMetrics
-                    ready: false
-                    name: nodemetrics
+          forEach:
+            properties:
+              - name: cpu
+                lookup:
+                  prometheus:
+                  - query: '1000 * max(rate(container_cpu_usage_seconds_total{container!="",node="${.component.name}"}[5m]))'
```

Comment:
This shouldn't be a forEach which is N+1, if you remove the node filter and return a map of lists then we only need to execute a single prometheus query